### DISCUSSION TITLE
chore(Html2Pdf):关于Table内使用pdf导出失败的问题

### DISCUSSION THREAD

**Discussion by shakugans** [dotnetcore/BootstrapBlazor#4508] [Q&A] [answered]
### Which class is this unit test associated with?

之前的某一个版本中pdf的导出是，最近做完另一个报表后测试导出发现失败了，回头查看之前的报表pdf导出也是失败
这个功能在官网测试是可用的，但把官网的项目下载下来运行Table导出或者其他Pdf导出也会失败
![1fc2bcde460273cc18b6fd3154fd7e87](https://github.com/user-attachments/assets/2ffad930-8c1d-45e5-9ed8-5e084100db71)
![9e7839594f610bb727555ac2678ccc6e](https://github.com/user-attachments/assets/d692fa7c-948f-4c8d-90e4-16095cfa6689)
可能是内网环境Chromium运行时下载失败的问题，但我不确定，因为我参考这个文章https://www.puppeteersharp.com/docs/ReuseChrome.html，在Program中加入了下载和启动的代码后依旧没有导出成功
![568fa01985981a524ef992496825ed34](https://github.com/user-attachments/assets/2981994e-c653-403e-a0b6-09ee22e24af5)
![991f03aca648d426252e29acc9aa7c90](https://github.com/user-attachments/assets/30a598ef-d6e8-4c09-901c-4a7d3632cdf6)
如果是AddBootstrapBlazorTableExcelExport服务弃用了的关系，那是否能提供一个保留那一版本导出的方法？

**Comment by bb-auto** [comment]
@shakugans Thank you for submitting. We will give feedback later.

**Comment by ArgoZhang** [comment, marked as answer]
@shakugans 换位思考一下，就发点截图，口述一下，实在不知道问题出在哪里，官网跟我项目上都是没有问题的。所以如果要快速解决问题，就需要提供复现工程，否则无法提供任何帮助

可能的问题

现在组件库内置了两个 pdf 导出组件服务

1. Puppeteersharp 需要科学上网下载 Chrome 浏览器内核
2. iText 这个是不需要的

组件库升级相对应的导出服务 `TableExport` 等包也要升级，请检查一下

**Comment by shakugans** [comment]
在做复现的时候发现可以了。应该是运行时自动下载完了。。

**Reply by ArgoZhang** [reply]
遇到问题尽量先做一下复现
